src/Middleware/Diagnostics.EntityFrameworkCore/src/MigrationsEndPointMiddleware.cs

```diff
@@ -132,8 +134,6 @@ private async Task InvokeCore(HttpContext context)
             return null;
         }
 
-        var contextType = Type.GetType(contextTypeName)!;
```

Comment:
Breaking change?

Reply:
Maybe not? The current code expects `contextTypeName` to match an exact FQN but doesn't use the resulting match: https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/Diagnostics.EntityFrameworkCore/src/MigrationsEndPointMiddleware.cs#L121-L135.